distribution/archives/build.gradle

```diff
@@ -151,6 +158,8 @@ distribution_archives {
     }
   }
 
+  // should really be `no-jdk-linux-s390x` as it ships without a JDK, however it seems that the build can't handle
```

Comment:
OCD nit if this is going to be a full sentence, it should start with a capital, e.g. "This should really ... .", otherwise remove the period at the end ;)

Reply:
yay, i'm not the only pedantic person out there 🥳 
i'll fix it when rebasing